ouroboros-consensus/test-storage/Test/Ouroboros/Storage/ImmutableDB/Model.hs

```diff
@@ -450,7 +457,10 @@ getEBBBinaryInfo :: (HasCallStack, MonadState (DBModel hash) m)
 getEBBBinaryInfo err epoch = do
     dbm@DBModel {..} <- get
     let currentEpoch = dbmCurrentEpoch dbm
-        inTheFuture  = epoch > currentEpoch || dbmTip == TipGen
+        inTheFuture  = epoch > currentEpoch ||
+          case dbmTip of
+            TipGen -> True
+            Tip _  -> False
```

Comment:
Mostly curiosity: What's the motivation for this change?

Reply:
As a prerequisite for 19fd30620de8dc134bbbba74fb4b880467c72ab6, we now track the hash of the block/ebb in the `Tip` (both in the impl and the model), so `dbmTip == TipGen` would require `Eq hash`, hence the simple pattern-match.